packages/backend/src/lib/ethereum.ts

```diff
@@ -5,9 +5,13 @@ import ERC20_ABI from './abis/ERC20.json';
 
 const { infuraId } = CONFIG;
 
-export const defaultProvider = new ethers.providers.InfuraProvider(1, infuraId);
+export const mainnetProvider = new ethers.providers.InfuraProvider(1, infuraId);
+export const polygonProvider = new ethers.providers.InfuraProvider(
+  137,
```

Comment:
Is there a constant we can use rather than a hardcoded int ?

Reply:
> Did you test the frontend for quests also?

Yeah, that is why we needed to create the Polygon provider. @tenfinney & I both successfully created quests.

Reply:
> Is there a constant we can use rather than a hardcoded int ?

We could make some… Something like:
```javascript
const chain = {
  polygon: 0x89,
  mainnet: 0x1,
}
```